### ISSUE TITLE
Support an optional description for each logged task

### ISSUE THREAD

**Issue by dchenbecker** [nuttycom/aftok#30]
As part of the timeline page, it would be helpful to be able to add a memo for work done to track what is happening. This information could then be exposed as part of the Overview page with reporting functionality (e.g. list logged activities by date and/or user)

**Comment by sellout** [comment]
Yeah, with this, I could potentially use Aftok as my time tracker for consulting work, which just then makes it easy to adopt the contributor/income division features down the road.
